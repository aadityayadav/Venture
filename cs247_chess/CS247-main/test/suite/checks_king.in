setup
+ K e4
+ k e5
done
- e5
+ k f5
done
- f5
+ k f4
done
- f4
+ k f3
done
- f3
+ k e3
done
- e3
+ k d3
done
- d3
+ k d4
done
- d4
+ k d5
done
- d5
+ k a8
done
