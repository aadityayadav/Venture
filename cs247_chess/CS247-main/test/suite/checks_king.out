The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
The initial board cannot have a player in check!
k_______
________
________
________
____K___
________
________
________
